apps/editing-toolkit/editing-toolkit-plugin/common/hide-plugin-buttons-mobile.scss

```diff
@@ -1,9 +1,9 @@
-@import '@wordpress/base-styles/mixins';
-@import '@wordpress/base-styles/variables';
-@import '@wordpress/base-styles/breakpoints';
+@use '@wordpress/base-styles/mixins';
+@use '@wordpress/base-styles/variables';
```

Comment:
Why are we importing `mixins` and `variables` if we're only using `breakpoints` in this file?